### ISSUE TITLE
AttributeError: Calling operator "bpy.ops.export_scene.obj" error, could not be found

### ISSUE THREAD

**Issue by gerthermel** [Zi9/Blender-CarX-Mod-Tools#5]
Python: Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\zi_cx_tools\MapTools.py", line 209, in execute
    bpy.ops.export_scene.obj(
  File "C:\Program Files\Blender Foundation\Blender 4.1\4.1\scripts\modules\bpy\ops.py", line 109, in __call__
    ret = _op_call(self.idname_py(), kw)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: Calling operator "bpy.ops.export_scene.obj" error, could not be found

**Comment by gerthermel** [comment]
By my understanding `bpy.ops.export_scene.obj` has been deprecated past 4.0 and you have to use `bpy.ops.wm.obj_export`.
I tried to use this...
```
            bpy.ops.wm.obj_export(
                filepath=filepath + ".obj",
                #use_selection=False,
                path_mode=context.scene.CX_ExpP.texexp,
            )
```
...but as I am not proficient in Python I don't know how far the "rabbit hole" goes, what else needs to be added/removed/changed

**Comment by Zi9** [comment]
Fixed in 668ef44ef1fa941a393469dd1a5b8e719a1bc8f3
